TweetBook/Program.cs

```diff
@@ -2,8 +2,11 @@
 global using TweetBook.Data;
 global using TweetBook.Services;
```

Comment:
While I understand why you have declared these ones as global, I have to mention that not **every single** class needs these imports. It is indeed an useful feature brought in C# 10, but we tend to avoid using it

Reply:
Good to know, thanks!